### PR TITLE
Display all types of messages in Warnings.

### PR DESCRIPTION
**Pull request by loopyme**
Working on #389

**About CI**: I run a trigger build, which failed while running test case:`tests/unit/test_example.py`. It is caused by `assert \"<span class=badge>10</span>\" in profile.to_html()` while the total number of warnings is 10 and all types of warnings are shown in my branch. 

Any ideas? Should I update the test case or abort this new functionality?

### CODE REVIEW

File: src/pandas_profiling/report/presentation/flavours/html/templates/warnings/warning_uniform.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<td>
+    <a class="anchor" href="#pp_var_{{ message.anchor_id }}"><code>{{ message.column_name }}</code></a> is uniformly
+    distributed
+</td>
+<td>
+    <span class="label label-warning">Constant</span>
```
Comment (sbrugman) [inline src/pandas_profiling/report/presentation/flavours/html/templates/warnings/warning_uniform.html:6]: Constant = Uniform

File: src/pandas_profiling/report/presentation/flavours/html/templates/warnings/warning_unique.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<td>
+    <a class="anchor" href="#pp_var_{{ message.anchor_id }}"><code>{{ message.column_name }}</code></a> has unique
+    values
+</td>
+<td>
+    <span class="label label-warning">Constant</span>
```
Comment (sbrugman) [inline src/pandas_profiling/report/presentation/flavours/html/templates/warnings/warning_unique.html:6]: Constant => Unique

**Comment by sbrugman** [comment]
> Working on #389
> 
> **About CI**: I run a trigger build, which failed while running test case:`tests/unit/test_example.py`. It is caused by `assert \"<span class=badge>10</span>\" in profile.to_html()` while the total number of warnings is 10 and all types of warnings are shown in my branch.
> 
> Any ideas? Should I update the test case or abort this new functionality?

You are right that this is causing the issue. The test should be updated, as the expected behaviour has changed: we now expect > 10 warnings.

**Comment by loopyme** [comment]
Sorry about the mistakes I make, I have fix them in my latest commit.

Besides, I keep ignoring the `Rejected` warnings, since `Rejected`=(`Unsupported` or `Constant`), display both of them seems redundant.

**Pull request closed without merging**